Search for libtirpc in $EBROOTGENTOO instead of /usr.
This is needed for --enable-xdr.

Author: Oliver Stueker
Date: 2024-10-28

--- configure.orig	2024-10-28 12:11:20.100977220 -0230
+++ configure	2024-10-28 14:11:26.262506710 -0230
@@ -47369,11 +47369,11 @@
 
             if test "x$enablexdr" = "xno"; then :
 
-              { $as_echo "$as_me:${as_lineno-$LINENO}: checking for XDR support in /usr/include/tirpc" >&5
-$as_echo_n "checking for XDR support in /usr/include/tirpc... " >&6; }
+              { $as_echo "$as_me:${as_lineno-$LINENO}: checking for XDR support in $EBROOTGENTOO/include/tirpc" >&5
+$as_echo_n "checking for XDR support in $EBROOTGENTOO/include/tirpc... " >&6; }
               old_CPPFLAGS="$CPPFLAGS"
               old_LIBS="$LIBS"
-              CPPFLAGS="$CPPFLAGS -I/usr/include/tirpc"
+              CPPFLAGS="$CPPFLAGS -I$EBROOTGENTOO/include/tirpc"
               LIBS="$LIBS -ltirpc"
 
 
@@ -47416,7 +47416,7 @@
 
                             if test "x$enablexdr" = "xyes"; then :
 
-                      libmesh_optional_INCLUDES="$libmesh_optional_INCLUDES -I/usr/include/tirpc"
+                      libmesh_optional_INCLUDES="$libmesh_optional_INCLUDES -I$EBROOTGENTOO/include/tirpc"
                       libmesh_optional_LIBS="$libmesh_optional_LIBS -ltirpc"
 
 fi
